en/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: news_post
+title: "Ruby 2.3.0-preview1 Released"
+author: "naruse"
+translator:
+date: 2015-11-11 14:00:00 +0000
+lang: en
+---
+
+We are pleased to announce the release of Ruby 2.3.0-preview1.
+
+Ruby 2.3.0-preview1 is the first preview of Ruby 2.3.0.
+Many new features and improvements are included.
+
+[Frozen String Literal
+Pragma](https://bugs.ruby-lang.org/issues/11473) is introduced. On
+Ruby 2.1, "str".freeze is optimized to reduce object allocation. Ruby
+2.3 introduces new magic comment and command line option to specify
```

Comment:
_Update, after installing and trying it out_ :)

maybe this would be clearer: "Ruby 2.3 introduces [...] to **freeze** all <del>the</del> string literals in the source files"

I still am not too clear about the relevance of the 2.1 paragraph.